### PR TITLE
[14.0][ADD] account_payment_view_check_number

### PR DESCRIPTION
**Pull request by Vimal-OSI**

### CODE REVIEW

**Review by patrickrwilson — APPROVED** [review]
Works great, LGTM

**Comment by patrickrwilson** [comment]
@dreispt check number pr for review.

File: account_payment_check_number/README.rst [hunk 1]
```diff
@@ -0,0 +1,20 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+============================
+Account Payment Check Number
+============================
+
+This module will manage check number on supplier payment screen.
```
Comment (dreispt) [inline account_payment_check_number/README.rst]:
```suggestion
This module makes the check number field visible in the Payments list.
```

File: account_payment_check_number/README.rst [hunk 2]
```diff
@@ -0,0 +1,20 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```
Comment (dreispt) [inline account_payment_check_number/README.rst]: Please move this into `readme/` subdirectory fragments

File: account_payment_check_number/views/account_payment_view.xml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<odoo>
+    <record id="view_account_supplier_payment_check_number_tree" model="ir.ui.view">
+        <field name="name">account.supplier.payment.check.number.tree</field>
+        <field name="model">account.payment</field>
+        <field name="inherit_id" ref="account.view_account_supplier_payment_tree" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='payment_method_id']" position="after">
+                <field name="check_number" />
```
Comment (dreispt) [inline account_payment_check_number/views/account_payment_view.xml]:
```suggestion
                <field name="check_number" optional="hide" />
```

File: account_payment_check_number/README.rst (hunk 1 above)
Comment (dreispt) [reply account_payment_check_number/README.rst]: Usage: "On the Payments list, the Check Number field is available as an option, using the far right icon in the list header."

**Review by dreispt — CHANGES_REQUESTED** [review]
See review comments.
I wonder if "account_payment_view_check_number" would be a better module name?

**Comment by Vimal-OSI** [comment]
@dreispt @patrickrwilson , all the suggested changes are done.

**Review by dreispt — APPROVED** [review]
Looks great!

**Pull request merged**